src/rard/templates/research/add_work_link.html

```diff
@@ -9,25 +14,37 @@
 
 {% block inner %}
 
-
-    {# Use a GET form to select the work and antiquarian #}
+    {# antiquarian form #}
     <form novalidate enctype="multipart/form-data" autocomplete='off' action='{{ request.path }}' class="form work-form" method='GET'>
         {% bootstrap_field form.antiquarian field_class='submit-on-change' %}
+        <span class="toggle-definite">
+        {% bootstrap_field form.definite_antiquarian %}</span>
+
     </form>
 
+    {# work form #}
     <form novalidate enctype="multipart/form-data" autocomplete='off' action='{{ request.path }}' class="form work-form" method='GET'>
         {% bootstrap_field form.work field_class='submit-on-change' %}
+        <span class="toggle-definite">
+        {% bootstrap_field form.definite_work %}</span>
         <input type='hidden' name='antiquarian' value='{{ antiquarian.pk }}' />
-    </form>
+        <input type='hidden' name='definite_antiquarian' {% if definite_antiquarian %} value={{ definite_antiquarian }} {% endif %} />
 
+    </form>
```

Comment:
Also, definite values aren't getting carried through when I click submit.

Reply:
This is unexpected; I'll get back to you